src/screens/ExchangeModal.tsx

```diff
@@ -683,7 +708,7 @@ export default function ExchangeModal({
       });
     }
 
-    const outputInUSD = multiply(outputPriceValue, outputAmount);
+    const outputInUSD = multiply(outputPriceValue!, outputAmount!);
```

Comment:
Why is this non-null asserted? Is TypeScript lying to us?

Reply:
oh

Reply:
It looks like only the first value `outputPriceValue` needs non null assertion as until `usePriceImpactDetails` those details are indeed undefined. However you wont be able to call the function its wrapped in which is the `handleSubmit`, so it shouldn't ever be undefined when that function is actually called but not sure TS can infer that logic